src/MouseAdapter.js

```diff
@@ -309,20 +352,231 @@ define(function(require) {
+               -1 if data appears to be decreasing.
+                0 if there is no discernable trend.
+         * @private
+         */
+        _calculateDeltaDirection: function(array, startAt, length, arrayMaxLength) {
+            arrayMaxLength = arrayMaxLength || array.length;
+            startAt = startAt || 0;
+            length = length || arrayMaxLength;
+            var total = 0;
+            var offsetIdx1,offsetIdx2;
+            for (var idx1 = 0; idx1 < length; idx1++) {
+                for (var idx2 = idx1+1; idx2 < length; idx2++) {
+                    offsetIdx1 = (idx1+startAt) % arrayMaxLength;
+                    offsetIdx2 = (idx2+startAt) % arrayMaxLength;
+                    if (array[offsetIdx1] < array[offsetIdx2]) {
+                        total += (1/(idx2-idx1));
+                    }
+                    if (array[offsetIdx1] > array[offsetIdx2]) {
+                        total += (-1/(idx2-idx1));
+                    }
+                }
+            }
+            if (total >= 1) {
+                return 1; // Increasing trend
+            }
+            if (total <= -1) {
+                return -1; // Decreasing trend
+            }
+            return 0; // Stable
+        },
+
+        /**
+         * This is a helper function for detecting user scroll events.  The scroll deltas
+         * will decay slowly over time, but they typically wont trend upwards again 
+         * unless the user performs another scroll.  Thus, when the scroll deltas are 
+         * trending downward and suddenly begin to increase, this is a strong indication 
+         * of a user event.  This function works by detecting these increases preceded by
+         * a normal decreasing trend.
+         *
+         * One advantage this method has is that it can detect nearly all user events.
+         * One disadvantage this method has is that it can be slow, owing to the fact 
+         * that it may take time (150ms+) to gather enough data before an upward trend is 
+         * discernable.
+         * @param {Array.<number>} array The array of data to analyze for this check
+         * @return {boolean} Returns true if conditions were satisfied to indicate the
+         * user performed a scroll event, else false
+         * @private
+         */
+        _detectDeltaIncrease: function(array) {
+            var firstRange = this._getDeltaArrayIndex(-5);
+            // Check the last 5 data points and determine if they are increasing
+            if (this._calculateDeltaDirection(array,firstRange,5,this.DELTA_ARRAY_SIZE) === 1) {
+                // Check the 5 data points prior to those and determine if they were decreasing
+                var secondRange = this._getDeltaArrayIndex(-10);
+                if (this._calculateDeltaDirection(array,secondRange,5,this.DELTA_ARRAY_SIZE) === -1) {
+                    return true;
+                }
+            }
+            return false;
```

Comment:
Well, they *could* be that simple, but i figured the tests would benefit more from real-life examples since they existed.  If somebody changes these functions, and they no longer work to detect *real* user interactions, it wont really matter if the tests still pass on the made-up numbers :P

I could add your simplified tests to supplement/explain the more complex tests.